cmd/sonobuoy/app/version.go

```diff
@@ -20,20 +20,76 @@ import (
 	"fmt"
 
 	"github.com/heptio/sonobuoy/pkg/buildinfo"
+	"github.com/heptio/sonobuoy/pkg/errlog"
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 )
 
-func init() {
-	RootCmd.AddCommand(versionCmd)
+type versionFlags struct {
+	kubecfg Kubeconfig
 }
 
-var versionCmd = &cobra.Command{
-	Use:   "version",
-	Short: "Print sonobuoy version",
-	Run:   runVersion,
-	Args:  cobra.ExactArgs(0),
+var versionflags versionFlags
+
+func init() {
+	cmd := &cobra.Command{
+		Use:   "version",
+		Short: "Print sonobuoy version",
+		Run:   runVersion,
+		Args:  cobra.ExactArgs(0),
+	}
+
+	AddKubeconfigFlag(&versionflags.kubecfg, cmd.Flags())
+
+	RootCmd.AddCommand(cmd)
 }
 
 func runVersion(cmd *cobra.Command, args []string) {
-	fmt.Println(buildinfo.Version)
+
+	fmt.Println(fmt.Sprintf("Sonobuoy Version: %s", buildinfo.Version))
```

Comment:
nit: Use `fmt.Printf("...%s\n")` rather than combining `Println` and `Sprintf`. Same for the lines below.